third_party/move/move-binary-format/src/deserializer.rs

```diff
@@ -1651,8 +1746,8 @@ impl SerializedType {
 #[repr(u8)]
 #[derive(Clone, Copy, Debug)]
 pub enum DeprecatedNominalResourceFlag {
-    NOMINAL_RESOURCE        = 0x1,
-    NORMAL_STRUCT           = 0x2,
+    NOMINAL_RESOURCE = 0x1,
```

Comment:
I wonder how these formatting changes came to be despite `rustfmt::skip`? Were these intentional? Same for `DeprecatedKind` below.

Reply:
No idea. I aligned them again.